basic_flutter/lib/data/network/employeelist/employee_list_content_response_dto.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:json_annotation/json_annotation.dart';
+part 'employee_list_content_response_dto.g.dart';
+
+@JsonSerializable()
+class EmployeeListContentResponseDto {
+  final int modificationCounter;
+  final int id;
+  final int employeeId;
+  final String name;
+  final String surname;
+  final String email;
+
+  EmployeeListContentResponseDto(this.modificationCounter, this.id, this.employeeId, this.name, this.surname, this.email);
```

Comment:
Named parameter would be better here, there are a lot of variables and in this way it is not so easy to understand.